orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -1387,6 +1387,37 @@ void RegisterTrainingOpSchemas() {
         }
       });
 
+  /**
+   * ClipGradNormInplace operator, taking multiple gradients as inputs (seq<tensor>).
+   * ClipGradNormInplace should be used in conjunction with optimizers that are expecting seq<tensor> gradients as input,
+   * since this op takes input a sequence of tensors and outputs a sequence of tensors there by avoiding the need
+   * for SequenceConstruct (and making any unnecessary copy).
+   * Please note that the gradient clipping happens inplace.
+   */
+  ONNX_CONTRIB_OPERATOR_SCHEMA(ClipGradNormInplace)
```

Comment:
nit: should we change this to InPlaceClipGradNorm to match other similar op names?